### DISCUSSION TITLE
App-Builder Layer aktivieren

### DISCUSSION THREAD

**Discussion by thomaskofler** [e-netze/webgis-community#146] [General]
Ich habe eine neue Portal-App gemacht (aktuelle Webgis-Version) und wollte Layer aktivieren, die beim Aufruf der App direkt angezeigt werden.  Schaut im App-Builder so aus:

![grafik](https://github.com/gis-eni/webgis-forum/assets/119412172/82cce2db-35a6-4ced-81fc-0ba8c35e55f4)

Veröffentlicht man dann die App, werden die Layer nicht angezeigt. Ich bilde mir ein, dass das schon Mal funktioniert hat. Aber vielleicht mache ich da einfach was falsch :)

**Comment by jugstalt** [comment]
Hallo Thomas,
das Funktioniert bei "Apps" nicht so wie bei Karten. Die Ansicht ist nur eine Vorschau. Einstellungen die du in der Vorschau triffst, werden nicht abgespeichert.
Ein "App" baut sich immer aus dem Template und den Parametern zusammen, die die in der Parameter Liste einstellst.

**Reply by thomaskofler** [reply]
Ok, kann man die Parameter so setzen, dass Layer beim Öffnen der App aktiviert sind?

**Reply by gis-eni** [reply]
Du kannst die App Templates (im portal/_templates) Verzeichnis theoretisch deinen Wünschten anpassen. Oder in dem Fall ein bestehendes Templete kopieren und die Anpassungen anbringen.

Was da zu tun ist, kommt immer auf deine Anforderungen an. Die Grundsätzliche Vorgehensweise haben wir letztens beim Side-by-Side Vergleich gezeigt:

siehe: https://github.com/gis-eni/webgis-forum/discussions/136
